app/src/main/res/layout/developer_options_fragment.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent">
+  <TextView
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="@string/developer_options"/>
+</LinearLayout>
```

Comment:
EOF missing.

Reply:
Done. And the TextView is just temporary, just to have a simple layout for DeveloperOptionsActivity. I have removed it and implemented the actual layout in the next PR #3331.

Reply:
Aah, you'll probably have to remove it from the PR. In general, we should not check in any temporary/wrong code in the codebase even if we've removed/corrected it in the stacked PR.

Reply:
Oh okay. I have modified the layout file. PTAL. Thanks.